templates/site/index.html

```diff
@@ -0,0 +1,209 @@
+                {% set user_dropdown_repeat_label_id = "timeframeDropdownRepeatLabel" %}
+                {% set user_datepicker_endson_id = "timeframeEndsOnDatepicker" %}
+                {% set dropdown_label_text = "Time Frame" %}
+                {% include "partials/dropdown-panel.html" %}
+            {% endwith %}
+
+            <div>
+                {% with %}
+                    {% set button_type = "button_location" %}
+                    {% set button_id = "showLocationMap" %}
+                    {% set button_label = "Location" %}
+                    {% include "partials/button.html" %}
+                {% endwith %}
+            </div>
+
+            {% with %}
+                {% include "partials/modal-map.html" %}
+            {% endwith %}
+
+            <div id="nonEnumQueryables" class="div-queryable-filters"></div>
+
+            <div id="dropdownQueryables" class="div-queryable-filters"></div>
+        </div>
+
+        </div>
+        <div class="content-search-right">
+            <div id="searchNavContainer" class="row div-search-nav-container">
+                <div class="content-search-title">
+                    <h5>Search <span id="titleSearchType" class="content-search-title-type"></span></h5>
+                </div>
+                <div class="content-search-description">
+                    <p class="body-1">
+                        Filter by date range to focus on specific timeframes,location to narrow down your search area,
+                        and keywords relevant to the type of imagery or dataset. Additional options, like cloud cover and
+                        sensor type, let you further customize your results to meet quality and relevance standards.
+                    </p>
+                </div>
+                <div id="searchFilterContainer" class="div-search-filter-container">
+                    <div id="mySearchFilters"> </div>
+                    <div id="fullFilterStringContainer" class="div-hidden-filter"></div>
+                    <div id="searchFilterButtonContainer">
+                        {% with %}
+                            {% set button_type = "button_reg_default_alt" %}
+                            {% set button_id = "searchFilterApply" %}
+                            {% set button_label = "Apply" %}
+                            {% include "partials/button.html" %}
+                        {% endwith %}
+
+                        {% with %}
+                                {% set button_type = "button_reg_default_alt" %}
+                                {% set button_id = "searchFilterClear" %}
+                                {% set button_label = "Clear" %}
+                                {% include "partials/button.html" %}
+                            {% endwith %}
+                        </div>
+                </div>
+
+                <div id="searchResultsNavigation" class="search-results-nav">
```

Comment:
Does not match the design in too many ways to list them... just go look at Figma again please.